scripts/dependency_report.sh

```diff
@@ -0,0 +1,126 @@
+#!/bin/bash
+#
+# Show any go dependencies in MicroShift's go.mod that do not appear
+# in other OpenShift components.
+
+set -o errexit
+set -o errtrace
+set -o nounset
+set -o pipefail
+
+REPOROOT="$(readlink -f "$(dirname "${BASH_SOURCE[0]}")/..")"
+WORKDIR="${REPOROOT}/_output/dependency_report"
+
+PULL_SECRET_FILE="${HOME}/.pull-secret.json"
```

Comment:
Let's accept this as an argument to the script?

Reply:
To work properly, it should also contain a key for registry.ci.openshift.org in addition to what we're getting in a pull secret. I think we might want to mention this in the usage message.

Reply:
Done